app/templates/view_agreements_g8.html

```diff
@@ -0,0 +1,48 @@
+{% import "toolkit/summary-table.html" as summary %}
+
+{% extends "_base_page.html" %}
+
+{% block breadcrumb %}
+  {%
+      with items = [
+          {
+              "link": url_for('.index'),
+              "label": "Admin home"
+          }
+      ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+  {% with smaller = True, heading = "{} framework agreements".format(framework.name) %}
+    {% include "toolkit/page-heading.html" %}
+  {% endwith %}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <div class="marketplace-paragraph">
+        <p>
+          Review the information for each agreement, accept it, and continue to the next one.
+        </p>
+        <p>
+          If there’s a problem with an agreement, contact the person who uploaded it and come back to it later. Ask
+          suppliers to resubmit from the link in the email they received after they were accepted on to the framework.
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <div class="grid-row">
+      <section class="column-one-third search-page-filters">
+
+      </section>
+      <section class="column-two-thirds" aria-label="Framework agreements">
```

Comment:
Update: after talking to @risicle the first column will have some content in it but this will be quite minimal. That being the case, it's still a good idea to remove the `aria-label` as it still won't be that hard to get to the search results.

That being the case we should also swap the `section` tags out for `div`s. `section` tags define regions in the document (see https://www.w3.org/TR/html-aam-1.0/#el-section) which we have established we don't need. `div`s have no role so can be used for visuals alone (see https://www.w3.org/TR/html-aam-1.0/#el-div).